clang/runtime/dpct-rt/include/dpct/codepin/codepin.hpp

```diff
@@ -26,18 +25,17 @@ inline void synchronize(cudaStream_t stream) { cudaStreamSynchronize(stream); }
 /// Generate API check point prolog.
 /// \param api_name The UID of the function call.
 /// \param stream The CUDA stream to synchronize the command execution.
-/// \param args The schema string and variable value pair list.
+/// \param args The var name string and variable value pair list.
 template <class... Args>
 void gen_prolog_API_CP(const std::string &api_name, cudaStream_t stream,
-                       Args... args) {
-  synchronize(stream);
-  dpct::experimental::detail::gen_log_API_CP(api_name, args...);
+                       Args... args) synchronize(stream);
```

Comment:
Missed left curly brackets
```suggestion
                       Args... args) {
                       synchronize(stream);
```